pallets/msa/src/signature_registry_tests.rs

```diff
@@ -0,0 +1,336 @@
+use crate::{
```

Comment:
Moved signature registry tests in here (except for replays) because of the need for a different config (mainly smaller signature registry limits) compared to what benchmarks needs.  This allowed me to undo changes in more or less unrelated tests to get them to pass as well.

Reply:
I wish there was a way to have some kind of runtime builder that you could pass in the config params and then you could use that runtime to run against tests. So far I have't seen anything like that if we can make one we can get rid of all these setup runtime duplicates